deployment/terraform/network.tf

```diff
@@ -13,5 +13,5 @@ module "vpc" {
   bastion_ebs_optimized      = var.bastion_ebs_optimized
 
   project     = var.project
-  environment = var.environment
+  environment = ""
```

Comment:
Same rationale for the empty string here as with the [database module](https://github.com/azavea/noaa-flood-mapping/pull/61/files#r471810529).